components/create/CreateNft.vue

```diff
@@ -0,0 +1,382 @@
+<template>
+  <div class="is-centered columns">
+    <Loader v-model="isLoading" :status="status" />
+    <form class="is-half column" @submit.prevent="createNft">
+      <CreateNftPreview
+        :name="form.name"
+        :collection="selectedCollection?.name"
+        :price="form.salePrice"
+        :symbol="chainSymbol"
+        :chain="currentChain"
+        :image="imagePreview" />
+
+      <h1 class="title is-size-3">
+        {{ $t('mint.nft.create') }}
+      </h1>
+
+      <!-- nft art -->
+      <NeoField :label="`${$t('mint.nft.art.label')} *`">
+        <div>
+          <p>{{ $t('mint.nft.art.message') }}</p>
+          <DropUpload
+            v-model="form.file"
+            required
+            expanded
+            preview
+            :label="$t('mint.nft.drop')" />
+        </div>
+      </NeoField>
+
+      <!-- nft name -->
+      <NeoField
+        :label="`${$t('mint.nft.name.label')} *`"
+        required
+        :error="!form.name">
+        <NeoInput
+          v-model="form.name"
+          required
+          :placeholder="$t('mint.nft.name.placeholder')" />
+      </NeoField>
+
+      <!-- nft description -->
+      <NeoField :label="`${$t('mint.nft.description.label')} (optional)`">
+        <NeoInput
+          v-model="form.description"
+          type="textarea"
+          has-counter
+          maxlength="1000"
+          height="10rem"
+          :placeholder="$t('mint.nft.description.placeholder')" />
+      </NeoField>
+
+      <!-- select collections -->
+      <NeoField :label="`${$t('mint.nft.collection.label')} *`">
+        <div>
+          <p>{{ $t('mint.nft.collection.message') }}</p>
+          <NeoSelect v-model="form.collections" class="mt-3" expanded required>
+            <option
+              v-for="collection in listOfCollection"
+              :key="collection.id"
+              :value="collection.id">
+              {{ collection.name || collection.id }} - ({{
+                collection.totalCount
+              }})
+            </option>
+          </NeoSelect>
+        </div>
+      </NeoField>
+
+      <!-- list for sale -->
+      <NeoField :label="$t('mint.nft.sale.label')" required>
+        <div class="w-full">
+          <p>{{ $t('mint.nft.sale.message') }}</p>
+        </div>
+        <NeoSwitch v-model="form.sale" :rounded="false" />
+      </NeoField>
+
+      <!-- list for sale price -->
+      <NeoField
+        v-if="form.sale"
+        required
+        :error="!form.salePrice"
+        :label="`${$t('price')} *`">
+        <div class="w-full">
+          <div
+            class="is-flex is-justify-content-space-between is-align-items-center">
+            <NeoInput
+              v-model="form.salePrice"
+              type="number"
+              placeholder="1 is the minimum"
+              :min="1"
+              expanded />
```

Comment:
what's this? why minimum 1?

Reply:
oh correct, no need to put `min=1` will remove it